src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $calendar-gap;
+
+  &__field {
+    @include field($size);
+    box-sizing: border-box;
+    border: $field-border solid black;
+    background-color: #eee;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    transition-duration: 0.5s;
+    transition-property: transform;
```

Comment:
you can also use transition - it is property that combines all possible `transition-something` properties 😉 

[Transition](https://www.w3schools.com/css/css3_transitions.asp)